### DISCUSSION TITLE
Many problems trying to reapply to the student github thing

### DISCUSSION THREAD

**Discussion by AmethystKnight** [community/community#107486] [GitHub Education]
### Hi everyone,

I got an email that my student thing had expired so i tried to reapply. Had many problems. And I don't know what to do. 

Initially tried to do it from the computer and had a lot of trouble there and then as well (this was a few weeks ago). I managed to get it done but then it got rejected with the sort of vague reasoning that they couldn't confirm my id or something. I interpreted this as they wanted me to sign up using my phone and take a picture of my student id directly from my phone. 

Except then i got stuck on an infinite loading screen when i clicked the button on the website to take a picture (it just showed the two circular arrows spinning infinitely). When i refreshed my page and tried again it had the same problem except now it says i have a pending request even though i never clicked to send that request and haven't taken my new photo yet. 

I don't know what to do. 
Github from what i can tell is not allowing people to contact them about this so idk what to do.

**Comment by emretufekci** [comment]
Many huge organizations are using sheerid to verify student/teacher. Github have to migrate to same approach. I've same problem with you, i've got student paper from the e-gov of Turkey however github is rejected even there's e-sign & qr-verification on the paper. 

I'll take my chance with taking photo of the student id but idk is it gonna work and it is ridiculous showing fresh-paper or logging into university e-mail is more secure way to verificate am i student. Student id had been given by two years ago...

**Comment by mw-xyz** [comment]
Wait for this new one to finish processing, then try using a different browser. Also try disabling any content blockers. Let me know if this helps
